README.md

```diff
@@ -17,3 +17,14 @@ This repo includes:
 ### Nightly builds
 
 The latest built and tested packages from this repo are available on this MyGet feed: https://dotnet.myget.org/gallery/aspnetwebstack-dev
+
+### Tags and releases
+
+Git tag|Git branch|Other products|MVC package versions|Web API package (product) versions|Web Pages package versions
+--------|--------------|------------|------------|------------|------------
+[v2.0.4](https://github.com/aspnet/AspNetWebStack/tree/v2.0.4)|[v2.0-rtm](https://github.com/aspnet/AspNetWebStack/tree/v2.0-rtm)||4.0.40804|4.0.30506|2.0.30506
+[v2.1](https://github.com/aspnet/AspNetWebStack/tree/v2.1)||ASP.NET and Web Tools 2012.2, VS 2012 Update 2 (not on http://nuget.org)|v4 2012.2 Update RTM|v1 2012.2 Update RTM|v2 2012.2 Update RTM
```

Comment:
Not thrilled with this row or the "Other products" column in general. We need to say *something* about the `v2.1` tag but I'm not sure details of the versions buried in WTE and VS matter. Thinking about removing everything in the 3 "package versions" columns and moving the "Other products" column to the end, perhaps naming it "Notes". Thoughts?